src/templates/blog-post.js

```diff
@@ -13,12 +13,14 @@ const BlogPostPageTemplate = ({ data, pageContext, location }) => {
 
   return (
     <Layout location={location} title={siteTitle}>
-      <SEO title={post.frontmatter.title} description={post.excerpt} />
+      <SEO title={post.frontmatter.title} description={post.frontmatter.summary} />
 
       <h2 css="text-align: center; margin: 32px 0 0; color: #005882;">{post.frontmatter.title}</h2>
 
       <small css="text-align: center;">{post.frontmatter.date}</small>
 
+      <p css="text-align: center;">{post.frontmatter.summary}</p>
```

Comment:
Don't know if we want the summary here as well or not 🤷‍♂️

Reply:
Let's leave it for now and see later on.